api/notd/manager.py

```diff
@@ -154,11 +65,13 @@ def __init__(self, blockProcessor: BlockProcessor, saver: Saver, retriever: Retr
 
     @staticmethod
     def get_sponsored_token() -> Token:
-        sponsoredToken = SPONSORED_TOKENS[0]['token']
+        sponsoredTokens = load_sponsored_token()
+        sponsoredToken = sponsoredTokens[0]['token']
```

Comment:
this doesnt look right - once this is a Sponsored token you shouldn't use dict semanticsa (`['token']`) it should be `sponsoredTokens[0].token`